### ISSUE TITLE
Auto-format code with Black. 

### ISSUE THREAD

**Issue by synchronizing** [Waultics/CryptoBook#25]
As discussed earlier in #23, this would be an excellent idea to ensure our code is auto-formatted on PR merge to master.

**Comment by synchronizing** [comment]
This would need to be a custom bash script for us to be able to instantly commit code. Good reference can be found [here](https://gist.github.com/MichaelCurrie/802ce28c993ff2dd632c).

**Comment by Alescontrela** [comment]
`CryptoBook-js` is tested for formatting in Travis CI. Shouldn't be that hard to implement with `CryptoBook-py`.

**Comment by synchronizing** [comment]
> `CryptoBook-js` is tested for formatting in Travis CI. Shouldn't be that hard to implement with `CryptoBook-py`.

Yeah, it would be a bash script away. It's not priority rn, so I'll just leave it as it is.
